lib_eio_linux/eio_linux.ml

```diff
@@ -1455,6 +1446,9 @@ let rec run : type a.
             )
           | Eio_unix.Private.Pipe sw -> Some (fun k ->
               let r, w = Unix.pipe ~cloexec:true () in
+              (* XXX workaround for issue #319, PR #327 *)
```

Comment:
It's not exactly a work-around, though. We'd prefer them non-blocking even without the bug, right? So there's nothing to fix, and no need for `XXX` here.

Reply:
Agreed, but it's the only case where we're setting a FD to nonblocking, I'd keep the issue mention at least.

Reply:
The mention of the reason is good, but get rid of the `XXX`. Otherwise people will wonder in future what they're supposed to be fixing here.

Reply:
fixed